src/main/java/org/sonar/plugins/stash/client/StashClient.java

```diff
@@ -111,31 +112,29 @@ public void postCommentOnPullRequest(PullRequestRef pr, String report)
     postCreate(request, json, MessageFormat.format(COMMENT_POST_ERROR_MESSAGE, pr.repository(), pr.pullRequestId()));
   }
 
+  public Collection<StashComment> getPullRequestOverviewComments(PullRequestRef pr) throws StashClientException {
+    return getPaged(
+        MessageFormat.format(API_ONE_PR + "/activities", baseUrl, pr.project(), pr.repository(), pr.pullRequestId()),

```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Format specifiers should be used instead of string concatenation. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS3457)